keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -0,0 +1,1762 @@
+-->
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input (including test refactors)
+- [ ] (R) Graduation criteria is in place
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+<!--
+This section is incredibly important for producing high-quality, user-focused
+documentation such as release notes or a development roadmap. It should be
+possible to collect this information before implementation begins, in order to
+avoid requiring implementors to split their attention between writing release
+notes and implementing the feature itself. KEP editors and SIG Docs
+should help to ensure that the tone and content of the `Summary` section is
+useful for a wide audience.
+
+A good summary is probably at least a paragraph in length.
+
+Both in this section and below, follow the guidelines of the [documentation
+style guide]. In particular, wrap lines to a reasonable length, to make it
+easier for reviewers to cite specific portions, and to minimize diff churn on
+updates.
+
+[documentation style guide]: https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md
+-->
+
+Users are increasingly deploying Kubernetes as management solution for new
+workloads (batch processing) and in new environments (edge computing). Such
+workloads no longer need just RAM and CPU, but also access to specialized
+hardware. With upcoming enhancements of data center interconnects, accelerators
+no longer need to be physically plugged into the nodes where they are
+used. Instead, accelerators can be added to nodes dynamically as needed.
+
+This KEP introduces a new API for describing which of these new resource kinds
+a pod needs. The API supports:
+
+- Network-attached resources. The existing [device manager API](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/resource-management/device-plugin.md)
```

Comment:
It's called Device Plugins in the [official documentation](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/). Let's use official name.

BTW, that proposal was archived, here is an updated link to it: https://github.com/kubernetes/design-proposals-archive/blob/main/resource-management/device-plugin.md